LukasChiama/Banka-App
# #165493605 Build out view transactions endpoint

#### What does this PR do?
Build endpoints to allow client and staff access to either a single transaction or all transactions respectively

#### Description of Task to be completed?

- Ensure GET requests to the endpoint `/api/v1/transactions/:id` is working

- Ensure GET requests to the endpoint `/api/v1/transactions/` is working

- Ensure clients can view their transaction details

- Ensure staff can view any transaction record and all transaction records

#### How should this be manually tested?

- Using Postman or Insomnia, make a GET request to either `/api/v1/transactions` or `/api/v1/transactions/:id`

- Provide a valid token in the Authorization headers

- If the token is a client token, ensure the transaction was carried out by the client making the request

- If the request is for a single transaction then provide the transaction ID

#### What are the relevant pivotal tracker stories?
[#165493605](https://www.pivotaltracker.com/story/show/165493605)

#### Screenshots (if appropriate)
![Screenshot (83)](https://user-images.githubusercontent.com/42539315/56509762-bbc2ad00-651f-11e9-9757-78c10cc3f05f.png)
